### PR TITLE
Fix nth-prime generator

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/nth-prime/NthPrimeTest.fs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This file was auto-generated based on version 1.0.0 of the canonical data.
+// This file was auto-generated based on version 2.0.0 of the canonical data.
```
Comment (robkeim) [inline exercises/nth-prime/NthPrimeTest.fs]: @ErikSchierboom do you know changed here? It's weird to see the change of a major version but no change to the rest of the file.

Comment (ErikSchierboom) [reply exercises/nth-prime/NthPrimeTest.fs]: Well, the generator was changed: https://github.com/exercism/fsharp/pull/423/files#diff-378e45c2fa29f82b16b9d312d5f5c18aR240 The type of field used to represent an error changed.

Comment (robkeim) [reply exercises/nth-prime/NthPrimeTest.fs]: Ok thanks for the clarification!

**Comment by robkeim** [comment]
Merged, thanks @ErikSchierboom!

**Pull request merged**